.github/workflows/typos.yaml

```diff
@@ -0,0 +1,10 @@
+name: Spell Check
```

Comment:
There are no plans to introduce this

Reply:
May I know why? I think this workflow can help reduce typo automatically.

Reply:
It's a little low on users at the moment, and we'll consider introducing it when it gets more users, or when the Kubernetes main repository uses it.

Reply:
I see. I'll close this PR and create a new PR to fix typo.

Reply:
https://github.com/kubernetes-sigs/kwok/pull/317